### DISCUSSION TITLE
Add e-mail template from file feature?

### DISCUSSION THREAD

**Discussion by DrPython3** [DrPython3/MailRipV2#3] [Ideas]
Hey there and thank you for visiting the Discussions section!

Today I just want to hear _**your opinion**_ on the following idea:

At the moment, the e-mail template for the delivery test is embedded in the code. This may lead to unnecessary extra work if someone wants to edit it for her or his personal needs. 

**_So shall I add a new feature to the code which allows to import any personal e-mail template from a txt-file?_**

That one within the code could remain as "default". Moreover, the import could include placeholders for more individual templates.

Thanks for your reply and btw: have a great year 2021!

Best regards,
DrPython3

**Comment by NoneAsName** [comment]
Is Good idea , some mail provider will block email if same text .  Please do that .

Second  :   Can you make it sort Good email by smtp , example .

smtp,office365.com.txt
smtp.gmail.com.txt

Same goes for others  ...

Lastly:  i still have issue with some combo.txt stopping halfway with error .  

thank you for the effort ..

**Reply by DrPython3** [reply]
Thank you for your feedback. Saving the results by smtp host is possible though I am a bit sceptic. In case the host has been found using MX records the name of the TXT might be very long ...

**Reply by DrPython3** [reply]
See "Release 5": e-mail template now available and hits are saved to separate files named like the certain smtp host. As expected, in some cases the filename will be annoying long though I have tested it successfully. Regarding your problem with the combo loader, I still need more information. Please see here: 

https://github.com/DrPython3/MailRipV2/discussions/2

because I close this topic.
